src/components/PeopleManagement/AddMemberTableAction.jsx

```diff
@@ -0,0 +1,13 @@
+import { Button } from '@openedx/paragon';
+import { Add } from '@openedx/paragon/icons';
+import PropTypes from 'prop-types';
+
+const AddMemberTableAction = ({ openModal }) => (
+  <Button iconBefore={Add} onClick={openModal} variant="outline-primary">Add members</Button>
```

Comment:
I'm not actually seeing this button as a table action? The only way I'm able to open up the add members modal is a weird stray edit icon on the Group Detail card after the "View group progress" button 
<img width="1354" alt="Screenshot 2024-12-12 at 2 51 56 PM" src="https://github.com/user-attachments/assets/6bcce786-1db3-4e86-93b5-b2eb79bad4c6" />